server/fleet/software.go

```diff
@@ -1,43 +1,16 @@
 package fleet
 
 import (
-	"fmt"
 	"time"
 )
 
-type CVE struct {
-	CVE         string `json:"cve" db:"cve"`
-	DetailsLink string `json:"details_link" db:"-"`
-	// These are double pointers so that we can omit them AND return nulls when needed.
-	// 1. omitted when using the free tier
-	// 2. null when using the premium tier, but there is no value available. This may be due to an issue with syncing cve scores.
-	// 3. non-null when using the premium tier, and value is available.
-	CVSSScore        **float64 `json:"cvss_score,omitempty" db:"cvss_score"`
-	EPSSProbability  **float64 `json:"epss_probability,omitempty" db:"epss_probability"`
-	CISAKnownExploit **bool    `json:"cisa_known_exploit,omitempty" db:"cisa_known_exploit"`
-}
-
-type CVEMeta struct {
-	CVE string `db:"cve"`
-	// CVSSScore is the Common Vulnerability Scoring System (CVSS) base score v3. The base score ranges from 0 - 10 and
-	// takes into account several different metrics.
-	// See https://nvd.nist.gov/vuln-metrics/cvss.
-	CVSSScore *float64 `db:"cvss_score"`
-	// EPSSProbability is the Exploit Prediction Scoring System (EPSS) score. It is the probability
-	// that a software vulnerability will be exploited in the next 30 days.
-	// See https://www.first.org/epss/.
-	EPSSProbability *float64 `db:"epss_probability"`
-	// CISAKnownExploit is whether the the software vulnerability is a known exploit according to CISA.
-	// See https://www.cisa.gov/known-exploited-vulnerabilities.
-	CISAKnownExploit *bool `db:"cisa_known_exploit"`
-	// Published is when the cve was published according to NIST.score
-	Published *time.Time `db:"published"`
-}
-
 // Must be kept in sync with the vendor column definition.
 const SoftwareVendorMaxLength = 114
 const SoftwareVendorMaxLengthFmt = "%.111s..."
 
+// TODO: Do we need this type?
```

Comment:
We might be implementing some interfaces with it... or we were some time ago, like a JSON deserialization or something for the db, can't recall.

Reply:
If we are not anymore, we can remove it, yeah.